src/main/java/eu/bbmri_eric/negotiator/service/BBMRIDiscoveryServiceClientImpl.java

```diff
@@ -235,7 +244,91 @@ private List<MolgenisCollection> findAllCollections() {
     } catch (WebClientResponseException | WebClientRequestException e) {
       log.warn(e.getMessage());
       log.warn("Molgenis is not reachable!");
-      return Collections.emptyList();
+      throw e;
```

Comment:
shouldnt there be som better handling of this excpetion?

Reply:
There could be but I woudn't waste time on that for the same reasons as before